### PR TITLE
Fix routes

### LINKED ISSUE #120: Bug: Fix Gem info routes
Currently requests like GET `/api/v1/gems/{gemname}.json` produce responses for gems, which names don't contain dash `-`. Requests fail if gem name contains dash, like with `file-tail-1.2.0.gem`

### PR DESCRIPTION
**Pull request by milkysoft**
close #120 

Fix routes to return valid gem info.

### CODE REVIEW

**Comment by milkysoft** [comment]
@g4s8 , this Pull request is very easy and urgent

File: src/main/java/com/artipie/gem/http/ApiGetSlice.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
     /**
      * Endpoint path pattern.
      */
-    public static final Pattern PATH_PATTERN = Pattern.compile("/api/v1/gems/([\\w]+).(json|yml)");
+    public static final Pattern PATH_PATTERN = Pattern.compile("/api/v1/gems/(.+).(json|yml)");
```
Comment (kirill-h5i) [inline src/main/java/com/artipie/gem/http/ApiGetSlice.java:56]: Why to accept any symbol if we need only `-` here?

**Review by kirill-h5i — CHANGES_REQUESTED** [review]
@milkysoft please check my comment

**Review by g4s8 — APPROVED** [review]
@milkysoft thanks

**Pull request merged**